src/plugins/auto/tests/unit/parse_meta_device_test.cpp

```diff
@@ -142,6 +149,7 @@ const std::vector<ConfigParams> testConfigs = {
 
     ConfigParams {"CPU(-1),GPU,VPUX",  {}, true},
     ConfigParams {"CPU(NA),GPU,VPUX",  {}, true},
+    ConfigParams {"INVALID_DEVICE",  {}, false},
```

Comment:
can you add INVALID_DEVICE,CPU, btw, why the second {} is empty here?

Reply:
Because an exception will be occurred as expected while getting property from INVALID_DEVICE.. An empty list of meta devices will be returned.

Reply:
btw, the second {} means nth because the 3rd parameter `true` means an exception will be expected.

Reply:
Covered by  [PR#17949](https://github.com/openvinotoolkit/openvino/pull/17949)